mp4.go

```diff
@@ -45,6 +47,438 @@ var atoms = atomNames(map[string]string{
 	"disk":    "disc",
 })
 
+var genreIDValues = map[int]string{
```

Comment:
Can you add a comment to point to the source of this list (for future reference).

Reply:
added a comment